src/simulation.rs

```diff
@@ -35,6 +38,12 @@ pub struct Context {
     pub syncing_users: Vec<OwnedUserId>,
     pub config: Arc<Config>,
     notifier: Sender<Event>,
+    pub user_notifier: Sender<UserNotifications>,
+    pub inworld_state: Arc<InWorldState>,
```

Comment:
I wouldn't introduce the concept of world yet, what if we keep just the channels instead?

Reply:
As we discussed, let's keep just a `channels` field since this repo is open source and we don't want to mix it with our implementation.